ccaas_builder/cmd/build/main.go

```diff
@@ -0,0 +1,163 @@
+package main
```

Comment:
I wanna some comments here, for this file.
Please correct me for any mistake.
This file looks like analysis the file with schema of `type Connection struct`.

Reply:
some additional comments are needed yes.

Reply:
@SamYuan1990 have updated the comments